app/layout.tsx

```diff
@@ -17,7 +17,7 @@ export default function RootLayout({ children }: { children: React.ReactNode })
       lang="en"
       data-theme={Theme.Default}
       className={cn(
-        'bg-level1-100 text--100 origin-top sm:scale[1] md:scale-[1.15] lg:scale-[1.4]',
```

Comment:
@buseca I needed to remove those classes from here because they messed up with positioning, position fix wasn't working properly when having scaled elements, you need to find another way to make things bigger at higher resolutions without playing with css transformations if possible :-D